.github/ISSUE_TEMPLATE.md

```diff
@@ -1,31 +1,28 @@
-## Summary
 
-*Briefly describe the issue and please DELETE the non applicable headings below. *
+### Observed behavior
 
-## System information
+description of the behavior that was observed, including screenshots or other references when applicable
 
-*Please specify the Kolibri version you were using and your operating system!*
+### Expected behavior
 
- - Version: ?
- - Operating system: ?
- - Browser: ?
+description of what behavior was expected but did not occur
 
-## Traceback or relevant snippet from server.log or browser console
+### User-facing consequences
 
-```
-INSERT TRACEBACK, LOG MESSAGES ETC. HERE
-```
+implications for a user of the application
 
-## How to reproduce
+### Errors and logs
 
-1. Steps
-1. To
-1. Reproduce
+```
+logs and errors here
```

Comment:
Suggesting:

    Errors and tracebacks from command line -- or relevant contents of ~/.kolibri/kolibri.log

Reply:
I tried to be more general since errors can come from the front or back-end. will elaborate